index.js

```diff
@@ -13,12 +13,13 @@ const inputDefaults = {
   'working-directory': '.',
 }
 
-export async function run() {
+export async function run(func = null) {
   try {
     const options = {}
     for (const key in inputDefaults) {
       options[key] = core.getInput(key)
     }
+    options['func'] = func
```

Comment:
After rebase, I think the changes above can be dropped

Reply:
Done